docs/index.rst

```diff
@@ -94,13 +100,17 @@ Zarr is a file storage format for chunked, compressed, N-dimensional arrays base
         Contributor's Guide
```

Comment:
Could see this point to the Developer's Guide to parallel the top-row.

Reply:
@joshmoore - I'm having trouble parsing your comment here. Can you restate it?

Reply:
The box at the bottom is labeled and links to "Contributor's Guide" while the link at the top is labeled and links to "Developer's Guide". It'd err on the side of using Dev Guide for both.